python/upload.sh

```diff
@@ -3,5 +3,6 @@
 # remember to update the README.rst file
 #pandoc --from=markdown --to=rst --output README.rst README.md
 #pandoc --from=markdown --to=rst --output CHANGELOG.rst CHANGELOG.md
-venv/bin/python setup.py sdist
+venv/bin/python -m pip install build
```

Comment:
Heh, might be a good idea to add `--upgrade` to this so it picks up any changes from the last time I ran it (as you can tell, sometimes it can be a while.)

I can add that later, though.